packages/ember/tailwind-vite/index.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
+import { css, js, packageJson } from "ember-apply";
+// eslint-disable-next-line n/no-unpublished-import
+import { execa } from "execa";
+// eslint-disable-next-line n/no-unpublished-import
+import fse from "fs-extra";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function run() {
+  await packageJson.addDevDependencies({
+    tailwindcss: "^3.0.0",
+    postcss: "^8.0.0",
+    autoprefixer: "^10.0.0",
+  });
+
+  await execa("npx", ["tailwindcss", "init", "-p"]);
+
+  await css.transform("app/styles/app.css", {
+    Once(root) {
+      root.append({ name: "tailwind", params: "base" });
+      root.append({ name: "tailwind", params: "components" });
+      root.append({ name: "tailwind", params: "utilities" });
+    },
+  });
+
+  await js.transform("tailwind.config.js", async ({ root, j }) => {
```

Comment:
This file name should have hhe .cjs extension, since it's a commonjs file.

Otherwise mjs if we want the module style config